spec/controllers/trainer_controller_spec.rb

```diff
@@ -65,6 +65,17 @@
         card = create_and_check_review_card(@user, @block, 5, 'house')
         expect(card.review_step).to eq(5)
       end
+
+      it 'set review_date step=1 levenshtein_distance=1' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.